src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Configurations/DefaultOpenApiConfigurationOptions.cs

```diff
@@ -83,7 +83,8 @@ protected static List<OpenApiServer> GetHostNames()
             }
 
             var hostnames = collection.Split(new[] { "," }, StringSplitOptions.RemoveEmptyEntries)
-                                      .Select(p => new OpenApiServer() { Url = p });
+                                      .Where(h => !string.IsNullOrWhiteSpace(h))
```

Comment:
As line 80-83 already checks the null value, this line is overkill.

Reply:
It's a different state. An input string has whitespace, it generated whitespace, like this.
* Input : "http://localhost, "
* Output: "http://localhost", " "

Reply:
Oh, you're right. Makes sense.